### ISSUE TITLE
Feature Contact counter or indication

### ISSUE THREAD

**Issue by christiano-git** [DP-3T/documents#220]
A “A Contact counter” or “Indication green Yellow, red.

I understand that this might be not in the scope. 
However based on my experience in the IT sector for protection and supervision area in Europe I understood “A user like to have a an indicator for 24/7 background application”.
There is a need to prove that the system is running. There is a wish to check with a simple use case, let say the phone of my girl friend and may self.  
Let say you have a motorcycle with ABS. Yes there is a ABS light on while you turn on the engine. However the first ride in spring many driver go on a parking space to prove the ABS.
From my point of view it would gain acceptance of this App and that would be of great value at low cost.

**Comment by lbarman** [comment]
Hi @christiano-git, thanks! If I understand correctly you want a visual way for users to confirm that a contact is happening "right now" to test the contact-tracing ? Sounds like a good idea!

**Comment by lbarman** [comment]
Hi @christiano-git, thanks! If I understand correctly you want a visual way for users to confirm that a contact is happening "right now" to test the contact-tracing ? Sounds like a good idea!

**Comment by christiano-git** [comment]
Yes something in this direction. I could think about 2-3 use case.

A) user would like to test with two device if approximate tracing is working. Indicate that there is handshake right now. 
This could create visibility and trust.

B) user would like two know the count of contacts during the last x days. Can be a count or a indicated low, medium, high.
This could create more awareness to keep physical distance

C) An indication on the main top screen that the App is on. Like the WLAN or VPN.
Make it visible that the App is on, as it will be used for weeks.


At the same time I agree to keep the app simple and robust

**Comment by lbarman** [comment]
Merging discussion from @davidbarnardwills https://github.com/DP-3T/documents/issues/91 who says the same thing but in a more general way `allow [users] to make sense of the result of the risk score (including all the possiblity of false positives etc)`

**Comment by kugelfish42** [comment]
+1 to having a feedback indication to the user that the system is working and doing something useful.

This could also be slightly gameified into keeping the contact count as low as possible.

**Comment by christiano-git** [comment]
I thought for a minimum implementation proposal a simplicity will help to keep the App robust.

Indicate date of tracing activation. This will give the user transparency that he is recording handshakes since a certain time.

Indicate date of last handshake. This will provide trust that’s App is working. User can test the proper working with two device by him self.
